tests/func/test_odb.py

```diff
@@ -230,27 +229,3 @@ def test_shared_cache(tmp_dir, dvc, group):
     }
 
     assert expected == actual
-
-
-def test_cache_dir_local(tmp_dir, dvc, caplog):
```

Comment:
This test is not related to `odb` at all. It is related to the cache, but this got renamed, so this no longer belongs here.